### PR TITLE
Implement copyFileToPod

### PR DESCRIPTION
**Pull request by elanv**
There are utilities copying files from a pod, but it is missing transferring files to a pod. Added a method transferring a file to a pod.

Issue #353

### CODE REVIEW

**Comment by elanv** [comment]
I signed it

**Comment by elanv** [comment]
I signed it

**Comment by elanv** [comment]
I added a unit test and the code changed a bit.

File: util/src/main/java/io/kubernetes/client/Copy.java [hunk 1]
```diff
@@ -161,4 +166,34 @@ public static void copyFileFromPod(String namespace, String pod, String srcPath,
...
+
+  public void copyFileToPod(
+      String namespace, String pod, String container, File srcFile, String destPath)
+      throws ApiException, IOException {
+    // Run decoding and extracting processes
+    final Process proc =
+        this.exec(
+            namespace,
+            pod,
+            new String[] {"sh", "-c", "base64 -d | tar -xf - -C " + destPath},
```
Comment (alexandrupetre) [inline util/src/main/java/io/kubernetes/client/Copy.java]: How will this work against a windows pod?

Comment (elanv) [reply util/src/main/java/io/kubernetes/client/Copy.java]: Sorry for the late reply. Currently this function has such a limitation. But basically the same is true for the "kubectl cp" issue, and it doesn't seem to be an easy one.

https://github.com/kubernetes/kubernetes/issues/58512

Comment (juyiwang7) [reply util/src/main/java/io/kubernetes/client/Copy.java]: My file is 120Mb I tried this code,and it returned at 17Mb. How can I resolve this problem?

Comment (auxten) [reply util/src/main/java/io/kubernetes/client/Copy.java]: > My file is 120Mb I tried this code,and it returned at 17Mb. How can I resolve this problem?

There may be a hole in your 120Mb file, you can test the file with `du -h your_file` to see the size without a hole.

**Pull request merged**